Monofoxe/Pipefoxe/SpriteGroup/SpriteGroupMathParser.cs

```diff
@@ -0,0 +1,228 @@
+﻿using System;
+using static Pipefoxe.SpriteGroup.SpriteGroupMathTokenizer;
+
+namespace Pipefoxe.SpriteGroup
+{
+	/// <summary>
+	/// This class parse the expresion and returns the number that corresponds to it
+	/// Might be slow since it uses recursion to parse the expresion
+	/// Not benchmarked yet
+	/// </summary>
+	//  Based on the code by Brad Robinson in this Medium Article https://medium.com/@toptensoftware/writing-a-simple-math-expression-engine-in-c-d414de18d4ce
+	public class SpriteGroupMathParser
+	{
+
+		public SpriteGroupMathParser(RawSprite sprite)
```

Comment:
Make this class static, there is no reason for it to be instantiated.

Reply:
Done =3